### ISSUE TITLE
Ability to share URLs using QR codes without 3rd party application

### ISSUE THREAD

**Issue by dirkcuys** [AgregoreWeb/markdown-blog#5]
Sharing QR codes using a 3rd party app during user testing proved to be tedious. The experience could be streamlined a lot by integrating generation of QR codes in the web app.

**Comment by RangerMauve** [comment]
Yeah, the initial goal was to integrate QR code generation into Agregore itself, but this might be good too.
